agent/command.go

```diff
@@ -124,6 +127,16 @@ func (a *Agent) runCommandSet(ctx context.Context, tc *taskContext, commandInfo
 				if options.isTaskCommands || options.shouldSetupFail {
 					return errors.Wrap(err, "command failed")
 				}
+				// TODO EVG-6009 remove the following once monitoring is done
```

Comment:
maybe make it critical?  what are you hoping to learn from this?

Reply:
This is to figure out what tasks may be relying on the fact that failures in pre are ignored. I'm doing this so that the log messages show in splunk